includes/coreclr-prefix.md

```diff
@@ -0,0 +1,2 @@
+> [!NOTE]
```

Comment:
Not sure if the doc team has any preference here, but if we are only including this text in one place it may be simpler to inline it there rather than using the include mechanism. The DOTNET version of this got included from many different pages.